configs/semi/faster_rcnn/semi_base_faster_rcnn_r50_caffe_fpn_180k_partial_coco.py

```diff
@@ -0,0 +1,49 @@
+_base_ = [
+    '../../_base_/models/faster_rcnn_r50_fpn.py',
+    '../../_base_/default_runtime.py', '../semi_partial_coco_detection.py'
+]
+
+model_wrapper = dict(
```

Comment:
model_wrapper? Where will it be built?

Reply:
`model_wrapper` will be replaced by `model` in https://github.com/open-mmlab/mmdetection/blob/3b72b12fe9b14de906d1363982b9fba05e7d47c1/mmdet/utils/replace_cfg_vals.py#L67

Reply:
mmengine has a similar cfg called `model_wrapper_cfg` which is for DDP wrappers.

Reply:
Need to add some comments about this logic in the config.